src/frontend/package.json

```
@@ -80,6 +80,7 @@
     "faker": "5.4.0",
     "fetch-mock": "9.11.0",
     "glob": "7.1.6",
+    "iframe-resizer": "^4.3.1",
```

We always use pinned dependency, it avoids some bad suprises 😅
```suggestion
    "iframe-resizer": "4.3.1",
```